app/schemas.py

```diff
@@ -463,7 +464,7 @@ class JobSchema(BaseSchema):
     processing_started = FlexibleDateTime()
     processing_finished = FlexibleDateTime()
 
-    job_status = field_for(models.JobStatus, "name", required=False)
+    job_status = auto_field()
```

Comment:
I see these values were removed. I'm assuming because we moved stuff out of the models into their own enum file. Was anything supposed to be passed back in here after the change?

Reply:
No, the auto_field() does what was needed.